compiler/src/iree/compiler/Codegen/SPIRV/test/pipeline_matmul_promotion.mlir

```diff
@@ -49,8 +49,8 @@ hal.executable @matmul_f32_128x256x64 {
   }
 }
 
-// CHECK-DAG: spirv.GlobalVariable @{{.+}} : !spirv.ptr<!spirv.struct<(!spirv.array<256 x vector<4xf32>>)>, Workgroup>
-// CHECK-DAG: spirv.GlobalVariable @{{.+}} : !spirv.ptr<!spirv.struct<(!spirv.array<1024 x vector<4xf32>>)>, Workgroup>
+// CHECK-DAG: spirv.GlobalVariable @{{.+}} : !spirv.ptr<!spirv.struct<(!spirv.array<1056 x vector<4xf32>>)>, Workgroup>
```

Comment:
Could you add a comment here to explain how the numbers are computed so that later it's obvious? Something like "The promotion itself requires 1024 vector<4xf32>. However, in order to avoid bank conflicts, it's padded into 1056 vector<4xf32>."